src/client/views/news.ejs

```diff
@@ -1,6 +1,11 @@
 <div id="content">                  
     <h1 class="center"><%=t('news.title')%></h1>
 
+    <p class="date"><%=t('news.july28-2024.date')%></p>
```

Comment:
I'd prefer if we left news announcements to me.. Also people don't see new news posts until I actually push an update to the website, so it throws off dates...

Reply:
At the time of a website update, I will provide a changelog.